src/OfflineWorkflowsSample/OfflineWorkflowsSample/Views/Browsing/PortalBrowserView.xaml

```diff
@@ -0,0 +1,55 @@
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:offlineWorkflowSample="using:OfflineWorkflowSample"
+    xmlns:converter="using:OfflineWorkflowSample.Infrastructure.Converter"
+    NavigationCacheMode="Required">
+    <Page.Resources>
+        <converter:NullEnumToBlankStringConverter x:Key="NullEnumToBlankStringConverter" />
+    </Page.Resources>
+    <SplitView DataContext="{StaticResource MainViewModel}"
+               IsPaneOpen="True"
+               DisplayMode="Inline"
+               PaneBackground="Transparent">
+        <SplitView.Pane>
+            <ScrollViewer>
+                <StackPanel Padding="5,0,20,0">
+                    <ListView ItemsSource="{Binding PortalViewModel.VisibleFolders}"
+                              SelectedItem="{Binding PortalViewModel.SelectedFolder, Mode=TwoWay}"
+                              Background="Transparent">
+                        <ListView.ItemTemplate>
+                            <DataTemplate x:DataType="offlineWorkflowSample:PortalFolderViewModel">
+                                <TextBlock Text="{Binding Title}" />
+                            </DataTemplate>
+                        </ListView.ItemTemplate>
+                    </ListView>
+                    <TextBlock Text="Filters" FontWeight="SemiBold" />
+                    <CheckBox IsChecked="{Binding PortalViewModel.OfflineOnlyFilter, Mode=TwoWay}"
+                              Margin="0,0,5,0"
+                              Content="Restrict to content available offline" />
+                    <SearchBox QueryText="{Binding PortalViewModel.SearchFilter, Mode=TwoWay}"
+                               PlaceholderText="Filter"
+                               Margin="0,5,0,5"
+                               SearchHistoryEnabled="False" />
+                    <ComboBox ItemsSource="{Binding PortalViewModel.AvailableTypeFilters}"
+                              HorizontalAlignment="Stretch"
+                              SelectedItem="{Binding PortalViewModel.TypeFilter,Mode=TwoWay}">
+                        <ComboBox.ItemTemplate>
+                            <DataTemplate>
+                                <TextBlock Text="{Binding Converter={StaticResource NullEnumToBlankStringConverter}}" />
+                            </DataTemplate>
+                        </ComboBox.ItemTemplate>
+                    </ComboBox>
+                </StackPanel>
+            </ScrollViewer>
+        </SplitView.Pane>
+        <SplitView.Content>
+            <GridView ItemsSource="{Binding PortalViewModel.SelectedFolder.Items}"
+                      ItemTemplate="{StaticResource PortalItemTemplate}"
+                      SelectedItem="{Binding Source={StaticResource MainViewModel},Path=SelectedItem,Mode=TwoWay}"
+                      Grid.Row="1"
+                      Margin="5,0,5,0" />
+        </SplitView.Content>
+    </SplitView>
```

Comment:
Use x:Bind instead of Binding

Reply:
Because selected item is bound to in multiple places, I don't think I can use x:Bind. When I made that change, the setter would be called first with the selected value, then two more times with the outdated value from the other bindings.